a.sh

```
@@ -0,0 +1,87 @@
+# Make sure to set required env vars to run:
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)